### PR TITLE
Feature/swift 4.1

### PR DESCRIPTION
**Pull request by akuzminskyi**
Updated the project to swift 4.1;

### CODE REVIEW

**Review by poksi592 — APPROVED** [review]

File: MDCNetworking/DynamicCoding.swift [hunk 1]
```diff
@@ -131,30 +131,87 @@ public extension KeyedDecodingContainer where Key == DynamicKey {
                 dict[key.stringValue] = value
```
Comment (poksi592) [inline MDCNetworking/DynamicCoding.swift:131]: AFAIK this should be removed by @bartlomiejn , right?

File: MDCNetworking/DynamicCoding.swift [hunk 2]
```diff
@@ -131,30 +131,87 @@ public extension KeyedDecodingContainer where Key == DynamicKey {
...
+                    dict[key.stringValue] = value
+                }
             }
         }
         
         return dict
     }
 }
+
+public extension UnkeyedDecodingContainer {
```
Comment (poksi592) [inline MDCNetworking/DynamicCoding.swift:169]: Regarding to the comment above, this change is unnecessary.

**Review by poksi592 — CHANGES_REQUESTED** [review]
Remove DynamicCoding, pls

File: MDCNetworking/DynamicCoding.swift (hunk 1 above)
Comment (akuzminskyi) [reply MDCNetworking/DynamicCoding.swift:131]: @poksi592  I've asked him, and we still use it.

**Pull request merged**